tests/unit/test_venv.py

```diff
@@ -1096,3 +1096,28 @@ def test_create_download(mocksession, newconfig, download):
     else:
         assert "--no-download" in map(str, args)
     mocksession._clearmocks()
+
+
+def test_path_append_prepend(tmpdir, mocksession, newconfig, monkeypatch):
+    config = newconfig(
+        [],
+        """\
+        [testenv:python]
+        commands=python -V
+        appendpath = /bin/append{:}
```

Comment:
I'd imagine this would make sense more with relative paths within toxinidir than absolute paths 🤔 otherwise the absolute paths will not make the workflow cross-platform, not?

Reply:
Right, or remove the first slash. Are forward slashes OK on Windows, or there is something like `{/}`?